vavr/pom.xml

```diff
@@ -61,6 +61,13 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <Automatic-Module-Name>io.vavr.core</Automatic-Module-Name>
```

Comment:
Please name it `io.vavr`. I read it is best practice to name the module like the main package of the project.

When we slice Vavr modules for version 1.0.0 we will have a module `io.vavr.core`.

Thanks!